app/src/main/java/de/westnordost/streetcomplete/quests/bench_backrest/AddBenchBackrest.kt

```diff
@@ -11,7 +11,7 @@ class AddBenchBackrest : OsmFilterQuestType<BenchBackrestAnswer>() {
 
     override val elementFilter = """
         nodes, ways with
-          amenity = bench
+          (amenity = bench or leisure = picnic_table)
```

Comment:
The question refers to benches. Is it going to be confused where asked for such cases?

Note that there is already answer that it is not a bench but rather a picnic table.

Reply:
As each picnic table (AFAIK) contains a bench, `Does this bench have a backrest?` sounded OK to me in English and Croatian.  You're correct though that `Other answer` of `It's a picnic table` is superfluous if we've asked the quest for `leisure=picnic_table`, however I wouldn't know how to programmatically disable that.

Reply:
With new SC method of asking questions, this is much clearer though.
```
Does this bench have a backrest?
Picnic table
```

![small_Screenshot_20220319_180830_de westnordost streetcomplete mn debug](https://user-images.githubusercontent.com/156656/159133897-22eeaf1a-f91b-40b4-bc2b-0399467f1a11.jpg)